SPECS/python-datadog/python-datadog.spec

```diff
@@ -0,0 +1,104 @@
+Name:           python-datadog
+Version:        0.44.0
+Release:        2%{?dist}
+Summary:        Python wrapper for the Datadog API
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/DataDog/datadogpy
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/datadogpy-%{version}.tar.gz
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-devel
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
+BuildArch:      noarch
+
+%description
+Datadogpy is a collection of tools suitable for inclusion in existing Python
+projects or for development of standalone scripts. It provides an abstraction on
+top of Datadog's raw HTTP interface and the Agent's StatsD metrics aggregation
+server, to interact with Datadog and efficiently report events and metrics.
+
+%package -n python%{python3_pkgversion}-datadog
+Summary:        Python wrapper for the Datadog API
+Requires:       python3-requests >= 2.6.0
+Provides:       python3-datadog
```

Comment:
This provides is unnecessary- that's already the name of the RPM, considering %{python3_pkgversion} is defined as `3`.

Reply:
Removed.